Elsewhen/Views/AltIconView.swift

```diff
@@ -8,13 +8,23 @@
 import SwiftUI
 
 struct AltIconView: View {
+    
+    #if !os(macOS)
+    @EnvironmentObject internal var orientationObserver: OrientationObserver
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass
+    #endif
+    
     @State var viewId = 1
     var done: () -> ()
     
-    var body: some View {
+    var isPadAndNotSlideOver: Bool {
+        DeviceType.isPad() && horizontalSizeClass != .compact
+    }
```

Comment:
Might want to make this a computed property on `DeviceType`?

Reply:
It relies on `horizontalSizeClass` though, which I'm not sure can be referenced from within `DeviceType`?

Reply:
On the 12.9" when you drag the middle slider and get then 2/3 & 1/3 (ish) splits, it's also a compact view - and the 50:50 split on an 11" is also compact ([ Multitasking Size Classes](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/#multitasking-size-classes)).

Reply:
> It relies on `horizontalSizeClass` though, which I'm not sure can be referenced from within `DeviceType`?

It can be accessed via `UIScreen.main.traitCollection.horizontalSizeClass`.

Alternatively, could be implemented as a protocol along the lines of:
```swift
protocol DetectsIpad: View {
    var horizontalSizeClass: Environment<UserInterfaceSizeClass> { get }
}

extension DetectsIpad {
    var isPadAndNotSlideOver: Bool {
        DeviceType.isPad() && horizontalSizeClass != .compact
    }
}
```

Reply:
@RosemaryOrchard Good point!
@dwrss learning something every day. Updated, though I didn't see much value in the protocol so I did it with a static computed property on `DeviceType`.